thorlcr/master/thmastermain.cpp

```diff
@@ -557,12 +557,17 @@ version: 1.0
   daliServers: dali
   watchdogEnabled: true
   watchdogProgressEnabled: true
+  cost:
+    thor:
+      master: "0.000002"
```

Comment:
this seems unrelated

Reply:
it's defaults from Shamer's stuff that was in the now deleted thor.xml